canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -868,10 +868,14 @@ Cypress.Commands.add("verifyNodeDoesExistInPaletteAtIndex", (nodeName, index) =>
 		.then((idx) => expect(idx).to.equal(index));
 });
 
-Cypress.Commands.add("verifyCategoryIsClosed", (categoryName) => {
+Cypress.Commands.add("verifyCategoryIsClosed", (categoryName) =>
+	!cy.verifyCategoryIsOpened());
+
```

Comment:
This is failing in the build.  You are not passing the `categoryName` parameter into the function call. However, even if you do that it will still fail.  It look like the way Cypress is set up it is not as easy as negating the returned value from `verifyCategoryIsOpened`.   So I recommend you change this back the way it was and use the  `		.should("not.exist");`

Reply:
Fixed.